### PR TITLE
refactor: Use logger instead of console

### PR DESCRIPTION
**Pull request by Crash--**
Cozy-Client can be used in a web app but also in konnectors or services. 

When run in the browser, there is no issue using console.* directly. But when it comes to run it on the server (services or server side konnectors), logs need to have a special format that cozy-client doesn't respect for now. See this issue https://github.com/cozy/cozy-client/issues/866 

The idea is to not use console.* anymore in CC, but use the logger. This is the first step. After that, we'll be able to format correctly the log.

### CODE REVIEW

**Review by Merkur39 — APPROVED** [review]
LGTM

**Review by trollepierre — COMMENTED** [review]
could we update `"no-console": 0,` into `"no-console": "error", // Use logger instead of console (see https://github.com/cozy/cozy-client/pull/1224 )`

File: packages/cozy-stack-client/src/logger.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import minilog from '@cozy/minilog'
```
Comment (paultranvan) [inline packages/cozy-stack-client/src/logger.js:1]: In services/connectors we use cozy-logger: https://github.com/cozy/cozy-libs/tree/master/packages/cozy-logger . Eventually, do we plan to use only one of them to log them all?

**Review by paultranvan — APPROVED** [review]
Very nice initiative :clap:

**Comment by Crash--** [comment]
> could we update "no-console": 0, into "no-console": "error", // Use logger instead of console (see https://github.com/cozy/cozy-client/pull/1224 ) inside .eslintrc.json?

I don't like that. There is few places where console.* are useful 

I prefer to be careful and let the dev think if the code will be used in production or not instead of forcing the use of logger  without understanding what is going on.

File: packages/cozy-stack-client/src/logger.js (hunk 1 above)
Comment (Crash--) [reply packages/cozy-stack-client/src/logger.js:1]: That's the goal at the end, to have only one logger that handle correctly browser & node. Though,  I don't know the path to do that

**Pull request merged**